### ISSUE TITLE
Add contact information for data services

### ISSUE THREAD

**Issue by brentryanjohnson** [18F/doi-extractives-data#2533]
With the integration of statistics.onrr.gov, we need a way for users to contact our data services team to request data that may not be available on the site. We'll also be redirecting users from statistics to NRRD. I'm hoping @coreycaitlin and @ericronne can help me think through design ideas to support contact info.

**Comment by coreycaitlin** [comment]
Options discussed with @brentryanjohnson:

- add to footer
- add to "About"
- add to "Downloads: Data and Documentation" page — this feels like the right first answer

On that page, we could create some kind of callout inviting people to contact Data Services if they don't find what they need. Ideally, this could be an include so that we can have this callout on all data/documentation pages.

**Comment by brentryanjohnson** [comment]
Contact information for data services:
- onrrdatarequests@onrr.gov
- 303-231-3162

**Comment by coreycaitlin** [comment]
Next steps:

- Discuss with @ericronne what this callout might look like
- Discuss with @msecret how to implement as an include (rather than as content we have to copy/paste onto multiple pages)

**Comment by ericronne** [comment]
I think About, Downloads, and footer all make good sense. Hopefully a lot of interested users will want to interact with us, about this and all sorts of other things. It seems a little odd that the only contact info we now have is a mailing address 🐢 

Let's chat!

**Comment by coreycaitlin** [comment]
For content to start with, let's try:

> If you need ONRR data that is not available here, contact Data Services with your request:
> 
> - onrrdatarequests@onrr.gov
> - 303-231-3162

**Comment by ericronne** [comment]
Here's a little module that can go almost anywhere necessary. I capriciously modded the text, to fit onto one line. Feel free to un-mod :)

![image](https://user-images.githubusercontent.com/9259185/34269750-1ffe9248-e64b-11e7-9ad3-645b0f92bda7.png)

**Comment by brentryanjohnson** [comment]
I think this is a good start, @ericronne. Why don't we start here and reassess based on user research as we go along. 

One minor edit...the email address should be: onrrdatarequests@onrr.gov

Any additional thoughts @coreycaitlin or @mentastc?

**Comment by ericronne** [comment]
Do we need "onrr" in the pre-@ part of the address, since it's the domain?

**Comment by brentryanjohnson** [comment]
I don't think the email address will resolve without it...we could look into simplifying the address with the ONRR IT folks.

**Comment by coreycaitlin** [comment]
Since it sounds like this is an existing email address that the Data Services team uses, let's include as-is.

**Comment by ericronne** [comment]
Oh i thought it was our concoction. If we know these people, it might be good to advise them.

![image](https://user-images.githubusercontent.com/9259185/34526392-f3ba4cd2-f067-11e7-8f3b-32bfa18d6e42.png)

**Comment by brentryanjohnson** [comment]
We can advise on a change down the road, but the existing contact information is on the [ONRR statistics site](https://statistics.onrr.gov/), which we're looking to fold into NRRD. That's the purpose of bringing it in as-is for now.

![image](https://user-images.githubusercontent.com/32855580/34528384-aa55489e-f05d-11e7-8a93-320465cbb104.png)

**Comment by ericronne** [comment]
styles …
![image](https://user-images.githubusercontent.com/9259185/34528747-76be672e-f06f-11e7-88e7-e951ad4e8830.png)

**Comment by coreycaitlin** [comment]
This is also a great opportunity for #2583

**Comment by brentryanjohnson** [comment]
@ericronne you mentioned a Sass variable to apply to that pipe...it looks like it's defined in this class: `<span class="header-nav_item_link_spacer"> | </span>`. Correct?

**Comment by ericronne** [comment]
Looks like that'll do the trick. It merely colors the object as $gray-light. We may want to rename that variable to something more generic.

**Comment by brentryanjohnson** [comment]
@ericronne On account of the _generous_ bottom margin for blockquotes, the contact information is pressed awkwardly against the "Revenue" subheading (ignore the missing icons for now).

![contact_paramd](https://user-images.githubusercontent.com/32855580/34787405-fe5223e8-f5eb-11e7-892f-70f3d4fde4f1.PNG)

---

An alternative is to style it as part of the blockquote, but that doesn't comport with the blockquote pattern in other instances and perhaps overemphasizes it's hierarchical importance (removing the bold would help with that).

![contact_blockquotestyle](https://user-images.githubusercontent.com/32855580/34787449-2955edf4-f5ec-11e7-9300-243c229c6abf.PNG)

---

Ideas?

**Comment by jennmalcolm** [comment]
Sorry to be late in the game on this, but the group is not called Data Services anymore. It is Data Retrieval. So, we need to change that in the text. Thanks for your progress on this issue @brentryanjohnson.

**Comment by coreycaitlin** [comment]
Currently blocked by #2601 

Next steps: @brentryanjohnson to discuss with @mentastc, change name from `Data Services` to `Data Retrieval`.

**Comment by brentryanjohnson** [comment]
We have multiple "data and documentation" pages with raw data and explanatory content. These include:

- [Main downloads](https://revenuedata.doi.gov/downloads/)
- [Federal Revenue by Location](https://revenuedata.doi.gov/downloads/federal-revenue-by-location/)
- [Federal Revenue by Company](https://revenuedata.doi.gov/downloads/federal-revenue-by-company/)
- [Reconciliation](https://revenuedata.doi.gov/downloads/reconciliation/)
- [Disbursements](https://revenuedata.doi.gov/downloads/disbursements/)
- [Federal Revenue by Location](https://revenuedata.doi.gov/downloads/disbursements/)

Only one of these pages makes use of the real estate to the right (on desktop) for navigation items. The others do not use the right column.

One idea @mentastc and I had is to use that real estate for a data retrieval contact include. While this approach departs from a UI pattern employed on most other pages, it does allow us to provide a consistent UI paradigm _within_ the download and doc pages, while providing persistent contact information while reviewing the content.

I've prototyped a page for review using the existing sticky-nav and container-right-3 classes with minor modifications to margins and width. Of course, we would need to refactor or create new classes for this, particularly in media queries for mobile and tablet.

![image](https://user-images.githubusercontent.com/32855580/35056096-bcb1a8bc-fb65-11e7-8d8c-76053afa942f.png)

**Comment by coreycaitlin** [comment]
I like this approach, and I think it makes some sense on these pages — but I'd also like to hear what @ericronne and @msecret think about it, since it breaks some existing patterns. Thoughts? worth doing?

It seems like the problem we're trying to solve here is making this snippet stand out for users who may be barely skimming the page, looking for contact info. Having this in the main column, styled similarly to the surrounding text, may not do enough to set it aside for users — particularly users who have already realized that the data doesn't include what they need. (That is, once users have decided this data isn't what they want, they're unlikely to read the documentation _more closely_, and more likely to start scanning _around_ the content for other pathways.)

**Comment by brentryanjohnson** [comment]
Agreed!

We also considered a scenario in which users might link to fragment ids within the page content, in which case persistent, sticky contact information would be in context with what users were viewing.

**Comment by ericronne** [comment]
If this info was sticky, it would be very prominent, and would likely generate a lot of clicks.

The sidebar positioning would only work on pages where there is no side nav, so that's a problem.

My gut would be to have it both in the footer and immediately following any content that might generate questions. Similar to the component concept above. Vertical spacing can be sussed out.

The footer currently lists only a mailing address. My guess (and it is a guess!) is that users might take that to mean, "We don't want to talk to you, but if you insist, send us a letter."

**Comment by brentryanjohnson** [comment]
Thank you, @ericronne. All good points.

My reservation about footer placement is that the data retrieval team isn't the point of contact for the website as a whole, which footer placement would imply. I agree, a postal address alone isn't especially useful.

Perhaps we can move forward with your guidance on placement following content that might generate data requests (data pages), and reassess after a round of user research.
